cmd/user-agent/src/pages/chapi/wallet/didcomm/blindedRouter.js

```diff
@@ -46,7 +45,13 @@ export class BlindedRouter {
 
         // send message to router requesting peerDID
         // TODO response peer DID from previous step to be sent to router
-        this.mediator.requestDID()
+
+        // TODO temporarily send myDID to test response (to test e2e)
+        let response = await _getPeerDID(this.aries, connection)
+        // ...
```

Comment:
`...` ???

Reply:
end of block, for TODO to be removed

Reply:
I wouldn't have known that unless you explained it... I would just remove it since we don't do that anywhere else... or add some sort of a "end temporary code" thing. Not really a big deal either way